packages/jest/src/matchers.ts

```diff
@@ -21,7 +21,10 @@ const kebabCase = (str: string) =>
 const removeSpaces = (str?: string) => str && str.replace(/\s/g, '');
 
 const mapProperties = (properties: Record<string, any>) =>
-  Object.keys(properties).map((property) => `${kebabCase(property)}:${properties[property]}`);
+  Object.keys(properties).map((property) => {
+    const key = property.startsWith('--') ? property : kebabCase(property);
```

Comment:
To be the devil's advocate here, it could technically break something, right?

Reply:
Maybe! I'll upgrade Jira and find out. It needs to match on both ends so I'm not too worried. If it was always kebab case - no break. If it was not kebab case, well, you just wouldn't be able to test anything because the matcher would never pass!